packages/studio/src/utils/downloadCode.js

```diff
@@ -1,7 +1,10 @@
-import { fileSave } from 'browser-fs-access'
+import { fileSave } from 'browser-fs-access';
+import builderAPI from './builderAPI';
 
-export default (code, fileName) => {
-  fileName = fileName ?? 'replicad-script'
+export default async (code, fileName) => {
+  fileName = fileName
```

Comment:
Are you sure this does something? I feel like it always take the name of the first export

Reply:
Would defaultName -> shapeName -> 'replicad-script` be a better order?

Reply:
Yeah, I think this would be a better order (as all shapes get a name by default).